docs/reference-guide/modules/events/pages/event-processors/streaming.adoc

```diff
@@ -1253,9 +987,9 @@ Hence, increasing the thread count should be paired with adjusting the segment c
 
 ==== Tracking processor threading
```

Comment:
We should drop the `TrackingEventProcessor` threading description entirely. The `PooledStreamingEventProcessor` description should be the only one left.